analysis/process.sh

```diff
@@ -0,0 +1,25 @@
+
```

Comment:
It should - I just took some lines out of your original notes.txt and put them here. There may be some minor modifications.